integration/tests/cook/test_cli.py

```diff
@@ -499,7 +499,8 @@ def test_list_by_state(self):
         waiting_uuid = uuids[0]
 
         # Submit a long-running job
-        cp, uuids = cli.submit('sleep 300', self.cook_url, submit_flags='--name %s' % name)
+        sleep_command = f'sleep {util.DEFAULT_TEST_TIMEOUT_SECS}'
+        cp, uuids = cli.submit(sleep_command, self.cook_url, submit_flags='--name %s' % name)
```

Comment:
This test fix is unrelated to the rest of the PR, but the test failed on this branch, so I fixed it here.